terracotta/drivers/sqlite.py

```diff
@@ -252,10 +253,19 @@ def get_datasets(self, where: Mapping[str, str] = None,
             if not all(key in self.key_names for key in where.keys()):
                 raise exceptions.InvalidKeyError('Encountered unrecognized keys in '
                                                  'where clause')
-            where_fragment = ' AND '.join([f'{key}=?' for key in where.keys()])
+            where = {
+                key: value if isinstance(value, list) else [value]
+                for key, value in where.items()
+            }
+            conditions = [
+                '(%s)' % ' OR '.join([f'{key}=?'] * len(value))
+                for key, value in where.items()
+            ]
+            values = list(itertools.chain(*where.values()))
+            where_fragment = ' AND '.join(conditions)
```

Comment:
Same here

Reply:
A delicate detail of style in code bases - the cases in which list comprehensions are used and where not. 😄I think we generally keep it down to very simple cases. Readability of the code is the property to optimize for.

For readability, I also prefer format strings over the old `%s` notation...